src/cli/cli.cpp

```diff
@@ -6216,15 +6247,37 @@ template <> otError Interpreter::Process<Cmd("region")>(Arg aArgs[])
 exit:
     return error;
 }
+    /**
+     * @cli releaserouterid (routerid)
+     * @code
+     * releaserouterid 16
+     * Done
+     * @endcode
+     * @cparam releaserouterid [@ca{routerid}]
+     * @par api_copy
+     * #otThreadReleaseRouterId
+     */
```

Comment:
Move this between 6261 and 6262 (between #if and template), and move left to align with "template"